googlemock/include/gmock/gmock-spec-builders.h

```diff
@@ -367,6 +367,15 @@ enum CallReaction {
 
 }  // namespace internal
 
+template <typename M>
```

Comment:
Circular dependency: these are defined in `gmock-generated-generated-nice-strict.h` which itself #includes `gmock-spec-builders.h`.